.azure-pipelines/client.yml

```diff
@@ -159,6 +166,6 @@ jobs:
       - task: PublishTestResults@2
         inputs:
           testResultsFiles: '**/test-results.xml'
-          testRunTitle: '$(os.name) Node $(node.version)'
+          testRunTitle: '$(OSName) Node $(NodeVersion)'
```

Comment:
Do you know if this title is interesting? In particular is it interesting to have the OSName included? If not just remove the OSName plumbing.

Reply:
I believe including the OS name in the test run title is required to determine which OS was used in the "test results" view

Reply:
Example: https://dev.azure.com/azure-sdk/public/_build/results?buildId=4873&view=ms.vss-test-web.build-test-results-tab

Reply:
That's correct. If it wasn't being used in the `testruntitle` I would have absolutely ripped the variable out of the strategy.

Reply:
Cool that is a good reason to me. We should update our other repo's as well to handle improve reporting.

Reply:
I wonder if there is a variable for the matrix strategy name that we could use for this as well.